client/protocols/hinting.py

```diff
@@ -167,7 +172,7 @@ def query_server(self, messages, test):
         request = urllib.request.Request(address)
         request.add_header("Content-Type", "application/json")
 
-        response = urllib.request.urlopen(request, serialized_data, 10)
+        response = urllib.request.urlopen(request, serialized_data, 60)
```

Comment:
1 minute is too long of a timeout. 30 - 35 seconds should be probably the upper limit.